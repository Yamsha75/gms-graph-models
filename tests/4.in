6 4 010101010
6 3 0110010011100110110101110
6 1 001001110
6 2 0110010101110110010101110
6 1 011101100001100001100000000001111010
6 4 011101110
6 1 011101110
6 2 0101101101011110
6 4 010111101011010000100010110100110000
6 2 011101110
6 3 0110
6 3 0110
6 3 0110
6 2 0111101011011010
6 3 0110
6 4 0110
6 3 0011000110110111110100110
6 2 0111110101110111010111110
6 4 0111101111011110
6 1 011101110
6 3 011101110
6 1 011111101111110111111011111100111100
6 4 0111110111110111110111110
6 1 0110110101110110010111110
6 2 010100100100000101111011000100001100
6 2 0110
6 1 0011001111001100
6 3 0111110100110011000010100
6 2 0111110111110111110111110
6 4 011101110
6 2 0111101111011110
6 1 001001000001100111001001001000111100
6 3 0110
6 2 011010101111110100011011110101010110
6 4 011101101111110111111010011101111010
6 2 001001110
6 4 0001001001011010
6 3 011101110
6 2 0110
6 4 0111110101110011000111110
