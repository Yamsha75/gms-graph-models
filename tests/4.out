5320
12133
4
432
24
3240
3
150
1733480
36
28
28
28
150
28
120
12133
348
32640
3
351
16
195000
12
1078
6
8
13106
300
3240
120
24
28
886
1154680
52
73968
351
6
293072
